src/Microsoft.Health.Dicom.SqlServer/Features/Store/SqlIndexDataStoreV6.cs

```diff
@@ -197,7 +198,7 @@ public override async Task<IEnumerable<VersionedInstanceIdentifier>> RetrieveDel
                             seriesInstanceUid,
                             sopInstanceUid,
                             watermark,
-                            partitionKey));
+                            new Partition(partitionKey, Partition.UnknownName)));
```

Comment:
This is incorrect, since the corresponding partitionKey will not match Unknown Name. Either we need to pass the name which we have from the requestcontext or remove the old files